### PR TITLE
Document Mapbox token provider

### PR DESCRIPTION
**Pull request by hackerstanislav**
JIRA: RAIL-4520

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1177/ : SUCCESS in 1m 20s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/785/ : SUCCESS in 6s

File: docs/02_start__using_boilerplate.md [hunk 1]
```diff
@@ -37,3 +37,10 @@ The created application contains its own README with further information about c
 **NOTE:** This tutorial assumes that you already have an existing GoodData platform or GoodData Cloud account or a GoodData.CN installation.
 You can create a GoodData platform account for free at [https://www.gooddata.com/free/](https://www.gooddata.com/free/), a GoodData Cloud account for free at [https://www.gooddata.com/trial/](https://www.gooddata.com/trial/) or download
 GoodData.CN at [https://www.gooddata.com/developers/](https://www.gooddata.com/developers/).
+
+## 3rd party cookies protection
+
+When you're using create gooddata react app, its necessary to turn of 3rd party cookies protections. Every browser has some
```
Comment (kandl) [inline docs/02_start__using_boilerplate.md:43]: Could we mention also "why" is this necessary?

File: docs/10_vis__geo_pushpin_chart_component.md [hunk 2]
```diff
@@ -72,12 +72,12 @@ const config = {
...
-| points | false | GeoPointsConfig | A configuration object where you can define clustering and the minimum and maximum sizes of the pins |
-| viewport | false | GeoConfigViewport | The region that the viewport should focus on after the chart is rendered |
-| tooltipText | false | [Attribute](50_custom__execution.md#attribute) | An additional item that shows a user-friendly label for the location attribute instead of the longitude and latitude |
+| Name | Required? | Type | Description                                                                                                                                                                                                                                                                                                                        |
+| :--- | :--- | :--- |:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```
Comment (kandl) [inline docs/10_vis__geo_pushpin_chart_component.md:76]: It's probably unnecessary to have `|:-------------`  so long?

File: docs/10_vis__geo_pushpin_chart_component.md [hunk 3]
```diff
@@ -144,3 +144,46 @@ The following example shows the supported `geoConfig` structure with sample valu
...
+## Example
+
+```jsx
+import "@gooddata/sdk-ui-geo/styles/css/main.css";
+import { GeoPushpinChart } from "@gooddata/sdk-ui-geo";
+import * as Md from "./md/full";
+
+<div style={{ height: 600, width: 900 }}>
+    <MapboxTokenProvider token="your_mapbox_token">
+		<GeoPushpinChart
```
Comment (kandl) [inline docs/10_vis__geo_pushpin_chart_component.md:174]: Formatting is somehow broken (mixed spaces and tabs)

File: docs/10_vis__geo_pushpin_chart_component.md [hunk 4]
```diff
@@ -144,3 +144,46 @@ The following example shows the supported `geoConfig` structure with sample valu
...
+			config={{}}
+		/>
+    </MapboxTokenProvider>
+</div>
+```
+
+## Properties
+
+| Name | Required? | Type | Description                                                                                                                                                                                                                                                                                                                        |
+| :--- | :--- | :--- |:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```
Comment (kandl) [inline docs/10_vis__geo_pushpin_chart_component.md:188]: Another looooong column. (see other columns)

File: docs/10_vis__geo_pushpin_chart_component.md [hunk 5]
```diff
@@ -144,3 +144,46 @@ The following example shows the supported `geoConfig` structure with sample valu
...
+import * as Md from "./md/full";
+
+<div style={{ height: 600, width: 900 }}>
+    <MapboxTokenProvider token="your_mapbox_token">
+		<GeoPushpinChart
+			location={Md.City.LOcation}
+			size={Md.Population.Sum}
+			color={Md.Density.Sum}
+			segmentBy={Md.StateName}
+			config={{}}
```
Comment (no23reason) [inline docs/10_vis__geo_pushpin_chart_component.md:179]: Should we leave the `config={{}}` out entirely?

File: docs/10_vis__geo_pushpin_chart_component.md (hunk 2 above)
Comment (no23reason) [reply docs/10_vis__geo_pushpin_chart_component.md:76]: This is done by prettier, we do not run it by default in the repo.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1188/ : SUCCESS in 1m 34s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/793/ : SUCCESS in 9s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1189/ : SUCCESS in 1m 04s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/794/ : SUCCESS in 8s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1193/ : SUCCESS in 1m 22s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/798/ : SUCCESS in 8s

**Review by ivanmjartan — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-gate-PR564

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1195/ : SUCCESS in 1m 46s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-fossa-scanning-zuul-el7/267/ : SUCCESS in 1m 05s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sonarqube-scan-zuul-gate/162/ : SUCCESS in 39s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/357/ : SUCCESS in 34s

**Pull request merged**